honggfuzz.BUILD

```diff
@@ -0,0 +1,176 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Bazel rules for building the Honggfuzz binary and the library linked with the
+# fuzz test executables.
+#
+# To use Honggfuzz, the following OS packages need to be installed:
+#   * libunwind-dev
```

Comment:
would it be possible to vendor or pull these in automatically somehow?

Reply:
Unfortunately I'm not aware of an easy way to do this with Bazel. I don't think these can be expressed as part of the dependency graph, unless we manually write BUILD rules for these packages to be built from scratch, from the source.

However, I do think we can make these more prominent. I'm planning to clean up our README file next, and I will include these in a "Prerequisites" section.